### ISSUE TITLE
Mechanism to synchronize slot transform updates

### ISSUE THREAD

**Issue by Lexevolution** [Neos-Metaverse/NeosPublic#3774]
### Describe the bug?

Specifically through any kind of photo (including inventory thumbnails), the position that specific meshes render at appear to be shifted.

### Relevant issues

Couldn't find any.

### To Reproduce

I don't know how to reproduce.

### Expected behavior

For the mesh to render in the correct position.

### Screenshots

Before issue: 
![image](https://user-images.githubusercontent.com/31176843/195986845-e9c61861-3645-455b-adda-4ab5b1d23d86.png)

After issue: 
![image](https://user-images.githubusercontent.com/31176843/195986853-73b58157-b35b-4ac5-af15-4d8fca3be668.png)

### How often does it happen?

Sometimes

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2022.1.28.1335

### What Platforms does this occur on?

Windows

### Did this work before?

I Don't Know

### Additional context

Sorry that this is a bad issue, but I'm creating it for the sake of adding to it in the future, hoping for more information. I believe having a bad issue is better than not reporting it at all.

### Reporters

Lexevo\#0876
ilcalmissimo\#1690

**Comment by LexiBasilisk** [comment]
Ah, this isn't an issue specific to photos but a photo does capture it well. Transforms are not always updated synchronously for every mesh under a slot (in this case, a user.) I've actually had several photos ruined by this too.

If you bake things together, this will not occur. Otherwise, it'll require a mechanism be added to batch transform updates together for a given slot.

Updating the issue name and label to reflect this.

**Comment by JackTheFoxOtter** [comment]
Yeah I've stumbled accross this as well, I had the beans as a separate mesh on my avatar and they sometimes weren't attached to my body in screenshots. This is actually a huge deal for me because it happens a lot, but to be fair I also need to optimize my avatar soon anyway ^^"
